src/tools/illink/src/linker/Mono.Linker.csproj

```diff
@@ -1,19 +1,42 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <AssemblyName>illink</AssemblyName>
     <Description>IL Linker</Description>
     <RootNamespace>Mono.Linker</RootNamespace>
+    <IsPackable>true</IsPackable>
```

Comment:
Make sure to add `<IsShipping>false</IsShipping>` so we don’t ship this package to NuGet.

Reply:
Seems like this package is being shipped to Nuget (https://www.nuget.org/packages/Microsoft.NET.ILLink), Im not sure if it's necessary for xamarin purposes though.

Reply:
If this package is already shipping, then we shouldn't mark it as non-shipping.

Reply:
That package should not be on public nuget.org and it was not until net8 p1. We are shipping it via internal feeds only, please remove it from nuget.org.

Reply:
cc @sbomer